<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_salledattente</class>
 <widget class="QDialog" name="dlg_salledattente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="MiseEnSalleDAttentegroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>161</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="title">
    <string>Retour en Salle d'attente  - Choisissez le statut</string>
   </property>
   <widget class="QRadioButton" name="RetourAccueilradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Examen terminé, en attente des informations de paiement</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ExamenEnAttenteAutreAvisradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Examen terminé, en attente d'être vu par</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ExamEnCoursradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Examen en cours, non terminé</string>
    </property>
   </widget>
   <widget class="QComboBox" name="UsercomboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="MsgtextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Msglabel">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="MessagetextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ExamEnCoursradioButton</tabstop>
  <tabstop>ExamenEnAttenteAutreAvisradioButton</tabstop>
  <tabstop>UsercomboBox</tabstop>
  <tabstop>RetourAccueilradioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
